Automattic/jetpack
# VideoPress: Upload video from dashboard

~### ⚠️ Needs #26406~

Allowing user to upload videos from Dashboard.

Closes #26316 

#### Changes proposed in this Pull Request:

* Extract `useUploader` out of blocks.
* Add a new default hook for `useUploader`.
* Include new `thumbnail` labels in `VideoGrid`.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `VideoPress`.
* Navigate to dashbord.
* Click `Add new video` (if already has videos) | or use drop/add first video (if has no videos).
* It should upload new video.

### Demo

https://user-images.githubusercontent.com/1663717/192428049-acc3165d-bd32-4e5e-b9f6-6e74fa405afc.mp4
